tutorials/parser_low_resource/test_parser_low_resource.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+from spacy.cli.project.assets import project_assets
+from spacy.cli.project.run import project_run
+
+
+def test_parser_low_resource_project():
+    root = Path(__file__).parent
+    project_assets(root)
+    project_run(root, "all", overrides={"vars.n_folds": 1, "vars.gpu": 0})
```

Comment:
The CI doesn't have a GPU:

```suggestion
    project_run(root, "all", overrides={"vars.n_folds": 1, "vars.gpu": -1})
```

(The tests are failing but this isn't being captured correctly by the new test script for some reason.)

Reply:
Actually, usually we use `-1` as the default GPU id for projects like this, so maybe that would be better here.